elements/pfe-icon/demo/index.html

```diff
@@ -125,8 +125,8 @@
 
   <style type="text/css" media="screen">
     body {
-      --pfe-icon--color: orange;
-      --pfe-icon--context: light;
+      /* --pfe-icon--color: orange;
```

Comment:
```suggestion
      /* Icon overrides for testing */
      /* --pfe-icon--color: orange;
```

^ A comment here might be helpful for future investigators